src/sth_database.js

```diff
@@ -142,8 +39,6 @@
     connectionURL = 'mongodb://' + authentication + '@' + dbURI + '/' + database +
       (replicaSet ? '/?replicaSet=' + replicaSet : '');
 
-    defineSchemas();
```

Comment:
Because we are using Mongoose only for the support of pool of connections to the database (see https://github.com/telefonicaid/IoT-STH/blob/feature/aggregation-of-text-occurencies/src/sth_database.js#L42). The complexity of the operations we are making on the database data and the flexibility we need for it makes it unpleasant to do it using Mongoose Models and Schemas... :)